elements/howto-menu/howto-menu.js

```diff
@@ -0,0 +1,115 @@
+/*
+ * A menu is a widget that offers a list of choices to the user,
+ * such as a set of actions or functions. A menu is usually opened,
+ * or made visible, by activating a menu button, choosing an item in a menu
+ * that opens a sub menu, or by invoking a command, such as Shift + F10 in
+ * Windows, that opens a context specific menu.
+ *
+ * The element that opens the menu is referenced with aria-labelledby.
+ * TODO(devnook): Should it be aria-labelledby or aria-describedby?
+ *
+ * See: https://www.w3.org/TR/wai-aria-practices-1.1/#menu
+ */
+
+ class HowtoMenu extends HTMLElement {
+
+   /**
+    * The constructor does work that needs to be executed _exactly_ once.
+    */
+   constructor() {
+     super();
+
+     this.KEY_CODES_ = {
```

Comment:
So far we’ve been putting our code in an IIFE and have the key codes outside the class ([example](https://github.com/GoogleChrome/howto-components/blob/f9f855beeaade5d3d01c0e9f91b861bbf092fe5b/elements/howto-radio-group/howto-radio-group.js#L28))